src/gluonts/mx/distribution/binned.py

```diff
@@ -70,6 +70,22 @@ def __init__(
     def F(self):
         return getF(self.bin_log_probs)
 
+    @property
+    def support(self) -> Tuple[Tensor, Tensor]:
```

Comment:
I think a better name for this would be `support_min_max` or `support_inf_sup`, given that for example [mxnet](https://github.com/apache/incubator-mxnet/blob/66488662ea6d33b8aeb64b632ea45194181f794a/python/mxnet/gluon/probability/distributions/gamma.py#L46) and [pytorch](https://github.com/pytorch/pytorch/blob/eb49dabe925b2fa32897dae771f831cb056aef65/torch/distributions/pareto.py#L48-L50) use `support` to provide the actual support